.github/workflows/linux.yml

```diff
@@ -6,14 +6,14 @@ on:
 
 jobs:
   Blint-GNU-Build:
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-latest
     steps:
-    - uses: actions/checkout@v3
+    - uses: actions/checkout@v4
       with:
         path: blint
-    - uses: actions/setup-python@v4
+    - uses: actions/setup-python@v5
       with:
-        python-version: '3.10'
+        python-version: '3.12'
```

Comment:
If the lowest is 3.9, then we set that here

Reply:
@prabhu Does this apply to Windows as well? I assume not since we use windows-latest and Windows doesn't ship with Python like linux does.

Reply:
Only applies to linux since it is a glibc specific limitation